bleak/backends/winrt/client.py

```diff
@@ -756,14 +760,22 @@ def bleak_callback(s, d):
         characteristic_obj = characteristic.obj
         if (
             characteristic_obj.characteristic_properties
-            & GattCharacteristicProperties.INDICATE
+            & GattCharacteristicProperties.NOTIFY
         ):
-            cccd = GattClientCharacteristicConfigurationDescriptorValue.INDICATE
+            if kwargs.get("force_indicate", False) and (
```

Comment:
Would it be simpler to just say:

    if device has notify property and not force_indicate:
        use notify flag
    else if device has indicate property
        use indicate flag
    else
        error